version.go

```diff
@@ -50,10 +54,16 @@ func AtLeast(major int64, minor int64) error {
 			return err
 		}
 	}
+
 	if version.Variant != "" {
-		log.Printf("non standard i3 payload variant '%s' detected. Ignoring version check. This is fully unsupported.", version.Variant)
+		versionWarning = true
```

Comment:
Remove this line: if you set versionWarning before checking it, the check never has any effect :)

Reply:
oh yep, just garbage from my previous attempt.